client/landing/gutenboarding/stores/domain-suggestions/index.ts

```diff
@@ -2,8 +2,9 @@
  * External dependencies
  */
 import { DomainSuggestions } from '@automattic/data-stores';
-import { getSignupDomainsSuggestionsVendor } from '../../utils/domain-suggestions';
+import { getSuggestionsVendor } from 'lib/domains/suggestions';
 
 export const DOMAIN_SUGGESTIONS_STORE = DomainSuggestions.register( {
-	vendor: getSignupDomainsSuggestionsVendor(),
+	/* Returns an ID for the domain suggestions vendor. Passing `true` to getSuggestionsVendor returns the signup variant.*/
+	vendor: getSuggestionsVendor( true ),
 } );
```

Comment:
I like removing the intermediate function, but it doesn't seem like material for this PR. Was it necessary?

Pulling out changes like this into independent PRs is a good way to keep the size down.

Reply:
It wasn't necessary, but it was in an effort to remove all duplicated code between the domain-picker package and gutenboarding.